test/trento_web/controllers/v1/host_controller_test.exs

```diff
@@ -13,31 +13,6 @@ defmodule TrentoWeb.V1.HostControllerTest do
   setup :setup_api_spec_v1
   setup :setup_user
 
-  describe "forbidden routes" do
```

Comment:
Let's remove this change. I have already done it in other PR, and it will conflict otherwise.

Reply:
@CDimonaco I mean, you didn't do any change here as far as I see, just move the `describe` to the bottom, which I did in the latest PR i merged some minutes ago